package.json

```diff
@@ -39,6 +39,7 @@
     "leaflet.markercluster": "^1.0.3",
     "lodash": "^3.9.3",
     "ng-file-upload": "^12.2.13",
+    "npm": "^6.1.0",
```

Comment:
there's no need for this here. `package.json` is an npm specification, so you would already have to have npm installed in order to interpret it.